src/utils/networkUtils.ts

```diff
@@ -44,24 +50,32 @@ export const SUPPORTED_WALLETS_MAP = {
 };
 export const SUPPORTED_WALLETS = Object.values(SUPPORTED_WALLETS_MAP);
 
-export const hasWeb3 = () => {
-	return window.web3;
-};
+export const hasWeb3 = () => !!window.web3;
 
-export const defaultNetwork = { name: 'MAINNET', networkId: 1 };
+export const defaultNetwork: { name: string; networkId: NetworkId } = {
+	name: 'MAINNET',
```

Comment:
should prob have an enum for the names here

Reply:
yeah all of that stuff should be in the snxjs lib